examples/next/pages/ui/components/ai/use-ai-generation/index.page.tsx

```diff
@@ -0,0 +1,46 @@
+// import { Amplify } from 'aws-amplify';
+// import { createAIHooks } from '@aws-amplify/ui-react-ai';
+// import { generateClient } from 'aws-amplify/api';
+// import '@aws-amplify/ui-react/styles.css';
+// import '@aws-amplify/ui-react-ai/ai-conversation-styles.css';
+
+// import outputs from './amplify_outputs.json';
+// import type { Schema } from './amplify/data/resource';
+// import { Authenticator } from '@aws-amplify/ui-react';
+// import React from 'react';
+
+// const client = generateClient<Schema>();
+// const { useAIGeneration } = createAIHooks(client);
+
+// Amplify.configure(outputs);
+
+// export default function Example() {
+//   const [{ data }, handler] = useAIGeneration('generateRecipe');
+//   const [_result, _handler2] = useAIGeneration('generateWhatever');
+//   return (
+//     <Authenticator>
+//       {({ user }) => {
+//         return (
+//           <>
+//             <h1>Hello {user.username}</h1>
+//             <div>{JSON.stringify(data)}</div>
+//             <button
+//               onClick={() => {
+//                 handler({
+//                   description:
+//                     'I want a recipe for a gluten-free chocolate cake.',
+//                 });
+//               }}
+//             >
+//               generate
+//             </button>
+//           </>
+//         );
+//       }}
+//     </Authenticator>
+//   );
+// }
```

Comment:
Why is this commented out here?

Reply:
This was commented out because the app didn't work without the envs, will fix as we begin working on the e2e tests!

Reply:
Perfect, thanks for the info!